flow/project.py

```diff
@@ -4242,6 +4249,16 @@ def _register_groups(self):
         for entry in group_entries:
             group = FlowGroup(entry.name, options=entry.options)
             self._groups[entry.name] = group
+            # Since we store aggregator parameter for operations as None,
+            # we fetch the aggregators associated with operations using
+            # the registered operations.
```

Comment:
I'm confused, is this comment related to the comment above? (quoted here)

```
        # We register aggregators associated with operation functions
        # in `_register_groups` and hence the aggregator parameter is passed as None.
        # This is because we do not restrict the decorator placement in terms of
        # `FlowGroupEntry`, `aggregator`, or `operation`.
```

Reply:
Yes, this is related to the quoted comment. I think of it as a hack to disable to placement requirements of the `aggregator` decorator.

Reply:
I don't know what to do about this immediately, but I think that it might be best to merge this as-is and simplify this logic later once we remove the legacy `add_operation` feature and consolidate group registration. Related discussion on PRs: #382, #487.

Reply:
@b-butler Also this comment is related to the above discussion of group/operation initialization.